scripts/release

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```

Comment:
Is there a reason this script and `try_release` are in a `scripts` directory, but `setup` is in `bin`? Do they have different situations/goals, or should they be combined if not?

Reply:
Oh maybe yeah let's combine. I'm just do used to the `./bin/setup` thoughtbot pattern.

Reply:
Cool, yeah! I'm good with `bin`; it's much easier to type than `scripts`. But whichever is fine, really.

Reply:
Ohh you mean the other way around? The `scripts` is a pretty strong npm convention cause the tool that runs them is called npm scripts.

Reply:
But we don't have to follow npm. I'll move to bin.

Reply:
Ha, sorry, I could've been more clear.

Looking at the original diff, I just didn't understand why some scripts were in `bin` and others in `scripts`. They were all shell scripts, so I wondered if they should all go together. But if npm has a strong convention, that's fine to keep them separated.